### DISCUSSION TITLE
What's the point of the togithub.com domain?

### DISCUSSION THREAD

**Discussion by oliversalzburg** [renovatebot/renovate#10796] [Request Help] [answered]
I noticed that some links in Renovate output use `togithub.com`, which apparently redirects to the same URL on `github.com`. Why?

**Comment by rarkins** [comment, marked as answer]
To avoid backlink and notification spam for other GitHub users and issues/PRs. There is no tracking or stats on them - simple 302s with no cookies IIRC

**Comment by oliviertassinari** [comment]
Dependabot uses `redirect.github.com` maybe a better option.

**Reply by HonkingGoose** [reply]
Thank you for the suggestion. I copied your suggestion into this issue:

- #29370

**Comment by jaisunislam1111** [comment]
.github/workflows/codespace-review-check.yml

**Comment by jaisunislam1111** [comment]
- ****

> ### ****
